battleship.py

```diff
@@ -71,7 +78,15 @@ def emptyGrid(rows, cols):
 Returns: 2D list of ints
 '''
 def createShip():
-    return
+    row = random.randint(1,8)
+    col =random.randint(1,8)
+    center =random.randint(0,1)
+    ship =[[],[],[]]

```

Comment:
No need to initialize ship in this way. Remove this line.

Reply:
Resolved the issue